README.md

```diff
@@ -12,62 +12,42 @@ Install from NPM:
 npm install @brightspace-ui/labs
 ```
 
-## README Index
-
-
-## Developing, Testing and Contributing
+## Developing and Contributing
 
 After cloning the repo, run `npm install` to install dependencies.
 
-### Linting
-
-```shell
-# eslint and stylelint
-npm run lint
-
-# eslint only
-npm run lint:eslint
-
-# stylelint only
-npm run lint:style
-```
-
 ### Testing
 
+To run the full suite of tests:
+
 ```shell
-# lint & run headless unit tests
 npm test
-
-# unit tests only
-npm run test:headless
-
-# debug or run a subset of local unit tests
-npm run test:headless:watch
 ```
 
-### Visual Diff Testing
+Alternatively, tests can be selectively run:
 
-This repo uses the [@brightspace-ui/visual-diff utility](https://github.com/BrightspaceUI/visual-diff/) to compare current snapshots against a set of golden snapshots stored in source control.
+```shell
+# eslint
+npm run lint:eslint
 
-The golden snapshots in source control must be updated by the [visual-diff GitHub Action](https://github.com/BrightspaceUI/actions/tree/main/visual-diff).  If a pull request results in visual differences, a draft pull request with the new goldens will automatically be opened against its branch.
+# stylelint
+npm run lint:style
 
-To run the tests locally to help troubleshoot or develop new tests, first install these dependencies:
+# translations
+npm run test:translations
 
-```shell
-npm install @brightspace-ui/visual-diff@X mocha@Y puppeteer@Z  --no-save
+# unit tests
+npm run test:unit
 ```
 
-Replace `X`, `Y` and `Z` with [the current versions](https://github.com/BrightspaceUI/actions/tree/main/visual-diff#current-dependency-versions) the action is using.
-
-Then run the tests:
+This repo uses [@brightspace-ui/testing](https://github.com/BrightspaceUI/testing)'s vdiff command to perform visual regression testing:
 
 ```shell
-# run visual-diff tests
-npx mocha './test/**/*.visual-diff.js' -t 10000
-# subset of visual-diff tests:
-npx mocha './test/**/*.visual-diff.js' -t 10000 -g some-pattern
-# update visual-diff goldens
-npx mocha './test/**/*.visual-diff.js' -t 10000 --golden
+# vdiff
+npm run test:vdiff
+
+# re-generate goldens
+npm run test:vdiff -- --golden
```

Comment:
Either works, though the official docs/help for `d2l-test-runner` would imply this is the "correct" way.
```suggestion
npm run test:vdiff golden
```